service/src/test/org/apache/hive/service/cli/TestHiveSQLException.java

```diff
@@ -18,13 +18,11 @@
 
 package org.apache.hive.service.cli;
 
-import java.util.List;
+import java.util.Collections;
```

Comment:
Is this import used?

Reply:
Excellent catch.  Will remove and then push.  Thanks.